common/python/Makefile

```diff
@@ -35,7 +35,7 @@ install:
 	pip3 install $(WHEEL_FILE)
 
 clean:
-	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
+	if pip3 uninstall --yes $(WHEEL_FILE); then echo UNINSTALLED WHEEL FILE ; fi
```

Comment:
Can we print which WHEEL file got uninstalled?
Applicable to all such occurances.

Reply:
done